db/migrate/20221026070207_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
```

Comment:
- [ ] In keeping with ruby conventions, I would recommend you use underscores for combined names instead camelcasing. So these attributes should be:

  ```
  t.integer :comments_counter
  t.integer :likes_counter
  ```